src/ServerConfig.cpp

```diff
@@ -216,3 +216,41 @@ ServerConfig& ServerConfig::operator=(const ServerConfig& copy)
     _servers = copy._servers;
     return *this;
 }
+
+// To populate the 'mimeTypes' variable
+std::map<std::string, std::string> createMimeTypes() {
+	std::map<std::string, std::string> mimeTypes;
+
+	mimeTypes[".pdf"] =  "application/pdf";
+
+	mimeTypes[".mp3"] =  "audio/mpeg";
+
+	mimeTypes[".jpeg"] = "image/jpeg";
+	mimeTypes[".jpg"] =  "image/jpeg";
+	mimeTypes[".png"] =  "image/png";
+	mimeTypes[".svg"] =  "image/svg+xml";
+
+	mimeTypes[".css"] =  "text/css";
+	mimeTypes[".html"] = "text/html";
+	mimeTypes[".htm"] =  "text/html";
+	mimeTypes[".js"] =   "text/javascript";
+
+	mimeTypes[".mp4"] =  "video/mp4";
+
+	return mimeTypes;
+}
+
+const std::map<std::string, std::string> ServerConfig::mimeTypes = createMimeTypes();
```

Comment:
We already have a MIME type table (ServerConfig::_mime), this is also populated with information from the config file. You might not have access to this in the server though. For now I will pass this because I will make the ServerConfig a singleton which should solve this issue (After which I will clean this up to use the correct variable).

Reply:
Can’t we have a global variable to store the server?

Reply:
Can’t we have a global variable to store the server?